### DISCUSSION TITLE
ASA Stats giveaways

### DISCUSSION THREAD

**Discussion by kerrilija** [asastats/channel#224] [General]
This discussion should be a sequel to [this](https://discord.com/channels/906917846754418770/914540797054025788/1017052295273398363):
> A bit off topic, but... What date is ASA Stats birthday? Every small giveaway gets a lot of attention

We should consider making random giveaways occasionally, for example when a milestone gets reached. They spread rapidly over social media.

Some of candidates(events):

- Mobile application launch
- 1st ASA Stats birthday (November 9th)
- The day when whitepaper was published (November 25th)

A free subscription giveaway to a random user could also be interesting.

**Comment by ipaleka** [comment]
Thank you @kerrilija, we should use this thread then for all the official giveaways.

On top of that, some ASASTATS are reserved in the Giveaway Pool for *random* giveaways. A half of those funds are reserved for the Team, while the other half is reserved to be spent by the community, From our [whitepaper](https://www.asastats.com/whitepaper.pdf):

> a giveaway can be conducted by any trusted community member, and even by using that member as a proxy (this means that a community member may send a giveaway from their account and ask for compensation from this pool afterward).

So, whenever you as the community members get an opportunity to meet an Algorand enthusiast who isn't yet familiar with ASA Stats and/or ASASTATS, we can send some ASASTATS to such a person. Three people have been rewarded from the Team part, and nobody, yet, from the part of the Giveaway Pool that belongs to the community.

**Comment by kerrilija** [comment]
Happy birthday ASA Stats nation!! :) Should we make a Twitter giveaway today? Let's say 300,000 ASASTATS?

**Reply by ipaleka** [reply]
> Happy birthday ASA Stats nation!! :) Should we make a Twitter giveaway today? Let's say 300,000 ASASTATS?

Happy birthday everybody! 

Yes, please, feel free to organize a giveaway in any way you want and with the amounts you find suitable. :)

**Reply by ipaleka** [reply]
> Let's say 300,000 ASASTATS?

Btw, is your suggested amount for a single winner or?

**Reply by kerrilija** [reply]
Initially it was. We could also go with a cap of 600,000 ASASTATS for three winners in 3:2:1 ratio. Suggestions and opinions are more than welcome.

**Comment by marucat27** [comment]
Great idea!  Any thoughts on how giveaway would be done?  Via twitter? Discord? Reddit?  I’ve seen projects ask users to tag 3 friends on twitter, like, and retweet for entry into contest.  Maybe could even have community submit what is their favorite feature of ASA Stats?

**Reply by kerrilija** [reply]
A Twitter post was in my mind. Participation in the giveaway will be secured with a like and retweet on the post. A random picker app would chose the winner. We could even record to prove it's random and post a video of the draft to our YouTube channel. Feel free to craft a post in #tweets-composing on our discord, or suggest another way we could do this: 

- single or multi winner

- the amount of ASASTATS to be allocated

- participation conditions

**Comment by kerrilija** [comment]
https://twitter.com/asastatscom/status/1591853851556061185

Twitter accounts of winners picked in the draw:

[PeterWelsh281.algo](https://twitter.com/PeterWelsh281) - 300,000 ASASTATS
[Intisan ☪️](https://twitter.com/Anucha_Intisan) - 200,000 ASASTATS
[TheFlowingRiver.shep 🐑](https://twitter.com/TheFlowingRiv3r) - 100,000 ASASTATS

Draw link:

https://twitterpicker.com/d/9sx3vba9

**Comment by kerrilija** [comment]
Whitepaper 1st anniversary Twitter giveaway:

https://twitter.com/asastatscom/status/1597727250945691649

Twitter accounts of winners picked in the draw:

[Legendary Tzu](https://twitter.com/bajiamun/) - 500,000 ASASTATS
[Think Big || Starknet Chimpion](https://twitter.com/larioflexwrap) - 400,000 ASASTATS
[WarabiHime](https://twitter.com/Fozabzy) - 300,000 ASASTATS
[Larry💎💎](https://twitter.com/official_Tolani) - 200,000 ASASTATS
[Komandar](https://twitter.com/Pacifist74) - 100,000 ASASTATS

Draw link:

https://twitterpicker.com/d/txmg6nbh

**Comment by kerrilija** [comment]
2022 FIFA World Cup Final match giveaway:

https://twitter.com/asastatscom/status/1604444134307233793

Twitter account of the winner (first guessed the correct result):

[Adewoye Tobi/MOO](https://twitter.com/AdewoyeTobi3) - 200,000 ASASTATS

**Comment by Leonazirite** [comment]
Discussions in ASA STAT has always been transparent.. No doubt the project remains and will always remain the legend in Algorand and beyond
